content/en/docs/setup/platform-setup/gke/index.md

```diff
@@ -36,6 +36,25 @@ Follow these instructions to prepare a GKE cluster for Istio.
     the `gcloud container clusters create` command.
     {{< /warning >}}
 
+    {{< warning >}}
```

Comment:
So it seems you are running into a generator bug. My guess is that the problem is caused by 3 levels of nesting: 1. numbered list item, 2. warning, 3. text section. If you have time and want to try to track down the problem, that would be great.

Otherwise, looking at your PR, I think the warning could be moved to a better place, anyway. You could probably work around the generator bug by moving the warning to the top or bottom of the doc, i.e., before of after the list of instructions (unindent to col 0), or just add it as another step (instead of a warning), something like:
```
1. **For private GKE clusters only**, an automatically created ...
```